test/api.js

```diff
@@ -0,0 +1,40 @@
+const got = require('got');
+const { expect } = require('chai');
+const nock = require('nock');
+const { default: defaultValues } = require('../lib/server/defaultValues');
+
+describe('return type check', () => {
+    it('Check 200 http and Name', async () => {
+        const firstExpectedId = 59791234;
```

Comment:
Please revert this test to the previous version. Currently you test only nock, not your application.